ui/raidboss/data/04-sb/raid/o10n.js

```diff
@@ -126,275 +126,130 @@
-        'Immortal Key': 'clef immortelle',
         'Midgardsormr': 'Midgardsormr',
       },
       'replaceText': {
-        'ready': 'prêt',
+        '(?<!/)Out': 'Dehors',
+        '(?<!\\w)In(?!/)': 'Dedans',
         'Akh Morn': 'Akh Morn',
         'Akh Rhai': 'Akh Rhai',
-        'Azure Wings': 'Ailes azur',
-        'Bloodied Maw': 'Gueule ensanglantée',
-        'Cardinals': 'Cardinaux',
         'Cauterize': 'Cautérisation',
-        'Coil': 'Charge',
         'Corners': 'Coins',
-        'Crimson Breath': 'Souffle écarlate',
-        'Crimson Wings': 'Ailes pourpres',
-        'Dark Wave': 'Vague de ténèbres',
         'Dry Ice': 'Poussière glaçante',
         'Earth Shaker': 'Secousse',
         'Exaflare': 'ExaBrasier',
-        'Flame Blast': 'Fournaise',
         'Flip': 'Tour vertical',
         'Frost Breath': 'Souffle glacé',
         'Horrid Roar': 'Rugissement horrible',
-        'Hot Tail': 'Queue calorifique',
-        '(?<!\\w)In(?!/)': 'In', // FIXME
         'In/Out': 'Dedans/Dehors',
-        '(?<!/)Out': 'Dehors',
         'Northern Cross': 'Croix du nord',
-        'Position': 'Position', // FIXME
         'Protostar': 'Proto-étoile',
-        'Rime Wreath': 'Enveloppe de givre',
-        'Shaker/Thunder': 'Secousse/Tempête',
-        'Signal': 'Signal', // FIXME
-        'Spin': 'Tour horizontal',
-        'Stygian Maw': 'Gueule ténébreuse',
+        'Spin': 'Gyrocoup',
         'Tail End': 'Pointe de queue',
         'Thunderstorm': 'Tempête de foudre',
-        'Time Immemorial': 'Big bang',
-        'Touchdown': 'Atterrissage',
-      },
-      '~effectNames': {
-        'Arcane Bulwark': 'Barrière magique',
-        'Crumbling Bulwark': 'Barrière magique détériorée',
-        'Death from Above': 'Désastre céleste',
-        'Death from Below': 'Désastre terrestre',
-        'Defenseless': 'Barrière magique bloquée',
-        'Landborne': 'Force terrestre',
-        'Skyborne': 'Force céleste',
-        'Thin Ice': 'Verglas',
+        'ready': 'prêt',
       },
     },
     {
       'locale': 'ja',
+      'missingTranslations': true,
       'replaceSync': {
         'Ancient Dragon': 'エンシェントドラゴン',
-        'Immortal Key': '竜の楔',
         'Midgardsormr': 'ミドガルズオルム',
       },
       'replaceText': {
-        'ready': 'ready', // FIXME
         'Akh Morn': 'アク・モーン',
         'Akh Rhai': 'アク・ラーイ',
-        'Azure Wings': '蒼翼の焔',
-        'Bloodied Maw': '紅牙の焔',
-        'Cardinals': 'Cardinals', // FIXME
         'Cauterize': 'カータライズ',
-        'Coil': 'Coil', // FIXME
-        'Corners': 'Corners', // FIXME
-        'Crimson Breath': 'クリムゾンブレス',
-        'Crimson Wings': '紅翼の焔',
-        'Dark Wave': 'ダークウェーブ',
         'Dry Ice': 'フリージングダスト',
         'Earth Shaker': 'アースシェイカー',
         'Exaflare': 'エクサフレア',
-        'Flame Blast': 'フレイムブラスト',
-        'Flip': 'Flip', // FIXME
         'Frost Breath': 'フロストブレス',
         'Horrid Roar': 'ホリッドロア',
-        'Hot Tail': 'ヒートテイル',
```

Comment:
Removing all of these lines makes me a little nervous.  How can you remove Hot Tail or Time Immemorial?

Reply:
As this is normal mode, this is never used (open js and txt file for o10n and search simply for hot, except the translation part, it is never used)